### PR TITLE
Create a [minimal] docker build environment for building lizmap packages.

### PR DESCRIPTION
**Pull request by dmarteau**
* Create a docker container with minimal setup to build package
* Create a makefile shortcut to build both environnement and lizmap package: `make -C dev build`
* Prepend '3liz/' to docker built image (i.e via `make docker-build-ci`)
* Prepend '3liz/' to locally docker built image tag (i.e when REGISTRY_URL is not defined) - this makes easier to run with [lizmap-docker-compose](https://github.com/3liz/lizmap-docker-compose)
* Fix ownership for files in docker images.

### CODE REVIEW

**Review by laurentj — APPROVED** [review]
LGTM. I tested on my machine and your container is working great. And I could verify that my patch fix the issue.

File: dev/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+from the `dev` directory
+```
+make build
+```
+
+or from the parent directory:
+```
+make -C dev build
+```
+
```
Comment (Gustry) [inline dev/README.md:14]:
```suggestion
A `build` directory is created with zip packages inside.

```

Just a quick note to explain what the user should check after running this command.
```

**Review by Gustry — APPROVED** [review]
Good to make a prefix with `3liz` !
I tried the command, it worked.

**Comment by nboisteault** [comment]
@dmarteau @laurentj I have this error : 
```
cd assets/ && npm install
npm WARN eslint-plugin-import@2.20.2 requires a peer of eslint@2.x - 6.x but none is installed. You must install peer dependencies yourself.
npm WARN assets@1.0.0 No repository field.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.1.3 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.13 (node_modules/watchpack-chokidar2/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.13: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

audited 765 packages in 4.209s

38 packages are looking for funding
  run `npm fund` for details

found 2 low severity vulnerabilities
  run `npm audit fix` to fix them, or `npm audit` for details

┌──────────────────────────────────────────────┐
│           npm update check failed            │
│     Try running with sudo or get access      │
│     to the local update config store via     │
│ sudo chown -R $USER:$(id -gn $USER) /.config │
└──────────────────────────────────────────────┘
cd assets/ && npm run build

> assets@1.0.0 build /build/assets
> webpack --config webpack.prod.js

Error: EACCES: permission denied, open '/build/lizmap/www/assets/js/lizmap.js'
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! assets@1.0.0 build: `webpack --config webpack.prod.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the assets@1.0.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /.npm/_logs/2021-02-03T08_27_10_312Z-debug.log
make: *** [Makefile:116: build] Error 1
Makefile:22: recipe for target 'build' failed
make: *** [build] Error 2
make : on quitte le répertoire « /home/nboisteault/3liz/infra/lizmap-docker-tests/dev »
```

**Comment by nboisteault** [comment]
@laurentj @dmarteau 
Sorry an error on my side.

**Review by nboisteault — APPROVED** [review]
I have tested. It is OK on my side too. Good job!

**Comment by laurentj** [comment]
I will commit more documentation when this PR will be merged

**Comment by nboisteault** [comment]
@dmarteau you can merge.

**Comment by nboisteault** [comment]
@dmarteau the backport action has failed. Can you backport to `release_3_4` manually?

**Comment by nboisteault** [comment]
@dmarteau gentle ping.

**Pull request merged**